<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchItem</class>
 <widget class="QWidget" name="SearchItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>368</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="item" native="true">
     <property name="styleSheet">
      <string notr="true">#item{
border-bottom: 1px solid rgb(211, 211, 211);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="title" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ClickableLabel" name="title_light">
           <property name="styleSheet">
            <string notr="true">#title_light:hover{
color: red;
}
#title_light{
font-family: &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>&lt;h3 style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:large; font-weight:600;&quot;&gt;好人有好报：&lt;span style=&quot; font-size:large; font-weight:600; color:#ff0000;&quot;&gt;抽象&lt;/span&gt;&lt;/span&gt;&lt;/h3&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="content" native="true">
        <property name="styleSheet">
         <string notr="true">#content QLabel{
color: rgb(108, 108, 108);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="content_label">
           <property name="font">
            <font>
             <family>微软雅黑</family>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#content_label{
font-family: &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥和这是啥&lt;span style=&quot; color:#ff0000;&quot;&gt;和这是啥和&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="tool" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#tool QLabel{
color: rgb(90, 90, 90);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="view_icon">
           <property name="text">
            <string>icon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tool_num">
           <property name="styleSheet">
            <string notr="true">#tool_num{
font-family: &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>123</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="classify_icon">
           <property name="text">
            <string>icon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tool_classify">
           <property name="styleSheet">
            <string notr="true">#tool_classify{
font-family: &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>默认分类</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="tool_time">
           <property name="styleSheet">
            <string notr="true">#tool_time{
font-family: &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>2022-07-20</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
